### ISSUE TITLE
Typo in script 3

### ISSUE THREAD

**Issue by majabedi** [BrunakSanderLabs/CancerRiskNet#1]
In the file scripts/Step3-CollectSearchResults.py line 31 refers to Step5 while not such a script exists.

**Comment by gkd720** [comment]
Yeah, I just started experimenting with this code and saw that too. I just changed it to "4" and it finishes as expected. But, a question. What is supposed to happen with its output. Should the script have thrown up some matplotlib plots? Or are the ".result" files to be used as input to some other program? It's a binary file, so is it just the raw image? What format?
